modules/acid/src/main/ts/ephox/acid/gui/components/SaturationBrightnessPalette.ts

```diff
@@ -104,7 +111,8 @@ const paletteFactory = (_translate: (key: string) => string, getClass: (key: str
       dom: {
         tag: 'div',
         attributes: {
-          role: 'presentation'
+          'role': 'presentation',
+          'aria-valuetext': 'Saturation 0%, Brightness 0%'
```

Comment:
Is this translated by a setAttribute call later, if that is the case remove this string. Serves no purpose. If not then translate this as well though the translate function with 0 and 0 as it's parameter inputs.

Reply:
No, I forgot to translate this one. That's on me.